yfinance/ticker.py

```diff
@@ -81,7 +81,7 @@ def _options2df(self, opt, tz=None):
         data['lastTradeDate'] = _pd.to_datetime(
             data['lastTradeDate'], unit='s')
         if tz is not None:
-            data['lastTradeDate'] = data['lastTradeDate'].tz_localize(tz)
+            data['lastTradeDate'] = data['lastTradeDate'].dt.tz_localize(tz)
```

Comment:
Hi, This is incorrect. Because the lastTradeDate is localized to utc only.

```
data['lastTradeDate1'] = pd.to_datetime(
    data['lastTradeDate'], unit='s', utc=True)
```

After localizing it to UTC. You then need to convert it to the correct timezone that you require
```
if tz is not None:
    data['lastTradeDate1'] = data['lastTradeDate1'].dt.tz_convert(tz)
```